docs/.vuepress/theme/styles/index.styl

```diff
@@ -136,6 +136,10 @@ body
       background-color var(--success-light)
       color var(--success)
 
+    &[muted]
```

Comment:
This CSS naming might be worth noting as well. "Muted" is for the buttons that are gray background with dark gray outline... as you may be able to tell from the CSS variables used below lol

Reply:
@benhaynes